crates/rethnet_rpc_server/src/lib.rs

```diff
@@ -39,7 +39,14 @@ pub enum MethodInvocation {
     Hardhat(HardhatMethodInvocation),
 }
 
-type StateType = Arc<RwLock<Box<dyn SyncState<StateError>>>>;
+type RethnetStateType = Arc<RwLock<Box<dyn SyncState<StateError>>>>;
+
+struct AppState {
+    rethnet: RethnetStateType,
+    fork_client: Option<Arc<Mutex<rethnet_eth::remote::client::RpcClient>>>,
```

Comment:
You can simplify this by using a `use` statement

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/3998/commits/63a0f5532e6ed156d79ff4cf3468575b69c93cd7